### PR TITLE
Add target user and slight reformat of UG

### LINKED ISSUE #173: [PE-D][Tester C] Difficult to read feature documentation
It would be good to differentiate the command format and the use cases of a command in different lines instead of locating them in the same section, like in the edit command section:

**Possible Fix:**
Separate them with a whitespace, or put them in different sections

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: jieqiboh/ped#6

### LINKED ISSUE #179: [PE-D][Tester D] Target user is not mentioned in the user guide
I think you should edit the default product description in the User Guide as it does not mention the target user.

I know the title is NUSContacts so it is meant to be used within NUS but I cannot tell if the product is meant for students, TAs, professors, or all of them to use. It's better to specify this as it's a marking point for the tP under Product Design.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: sindhurajain/ped#4

### PR DESCRIPTION
**Pull request by LargeCrowd**
Closes #173
Closes #179

### CODE REVIEW

**Comment by LargeCrowd** [comment]
Included value proposition, closes #156

File: docs/UserGuide.md [hunk 1]
```diff
@@ -122,25 +126,18 @@ Examples:
...
+| `t/TAG` | Tags of the contact. Similarly, `t/` with an empty tag is not supported. If you want to not include any tag, leave out `t/TAG` entirely.                                                                                                                                                                                         | `t/friend`             | Yes |
+| `f/` | Bypasses validation. Several fields have some validation rules (for example, you cannot use `ABCD` as a course code, since it does not conform to NUS course code format). Nevertheless, if you enter such a course code as input, the program will allow you to bypass the validation by adding `f/` to the end of the command. | `f/`                   | Yes |
+
+  :bulb: Note that `f/` must come at the end, or immediately before a tag. For example, `add f/ n/Alice …` is allowed, but `add
```
Comment (reetmitra) [inline docs/UserGuide.md:140]: I change this line alr so should reflect after merging.

**Review by reetmitra — APPROVED** [review]
I think looks okay for whatever has been added.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -391,13 +390,13 @@ For example:
...
-| [**Help**](#viewing-help-help)   | `help`                                                                                                                                                                          |
-| [**Exit**](#exiting-the-program-exit)   | `exit`                                                                                                                                                                          |
-| [**Copy Email**](#copying-a-contacts-email-to-clipboard-copy)   | `copy INDEX`                                                                                                                                                                          |
-| [**Undo**](#undoing-the-last-command-undo)   | `undo`                                                                                                                                                                          |
+| **Add**    | `add n/NAME e/EMAIL r/ROLE a/ADDRESS c/COURSE [t/TAG]… [p/PHONE]` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com r/STUDENT a/PGPR c/CS2103T t/friend t/colleague` |
```
[the remark this answers is not in the document]
Comment (LargeCrowd) [reply docs/UserGuide.md:393]: Oh i didnt mean to delete that, i think it was from a merge commit. let me revert this

[the remark this answers is not in the document]
Comment (LargeCrowd) [reply docs/UserGuide.md]: Please check again, thank you

File: docs/UserGuide.md [hunk 3]
```diff
@@ -152,10 +149,12 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/ROLE] [a/ADDRESS] [c/COURSE]
...
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the contact will be removed i.e. adding of tags is not cumulative.
-* You can remove all the contact’s tags by typing `t/` without
-    specifying any tags after it.
-* You can remove a contact's phone number by typing `p/` without specifying any phone number after it.
+* When editing tags, the existing tags of the person will be removed i.e. adding of tags is not cumulative.
+
+<div markdown="span" class="alert alert-primary">
+:bulb: **Tip:** You can remove all the person’s tags by typing `t/` without specifying any tags after it.
+:bulb: **Tip:** You can remove a person's phone number by typing `p/` without specifying any phone number after it.
```
Comment (E0735389) [inline docs/UserGuide.md:156]: Please run Jekyll server locally and check if things render correctly before pushing. Now this lightbulb got on the previous line, which I think is not intended.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -122,25 +126,18 @@ Examples:
...
+| `p/PHONE` | Phone number of the contact. Unlike the `edit` command, `p/` without any phone number is not supported. If you want to not specify the phone number, leave out `p/PHONE` entirely.                                                                                                                                               | `p/98765432`           | Yes |
+| `t/TAG` | Tags of the contact. Similarly, `t/` with an empty tag is not supported. If you want to not include any tag, leave out `t/TAG` entirely.                                                                                                                                                                                         | `t/friend`             | Yes |
+| `f/` | Bypasses validation. Several fields have some validation rules (for example, you cannot use `ABCD` as a course code, since it does not conform to NUS course code format). Nevertheless, if you enter such a course code as input, the program will allow you to bypass the validation by adding `f/` to the end of the command. | `f/`                   | Yes |
```
Comment (E0735389) [inline docs/UserGuide.md:138]: Last column looks confusing. Better having the title "Required?" and each cell either "Required" or "Optional".

**Pull request merged**